services/db/migrations/20170906142816_trip.js

```diff
@@ -1,8 +1,8 @@
 const setup = knex =>
   knex.schema.createTable('trip', (t) => {
     t.increments().index();
-    t.integer('tracknum');
-    t.enu('status', ['valid', 'invalid']);
+    t.text('tracknum');
```

Comment:
I think you can use `.unique` for this since each tracknum is supposed to be a unique value.

Reply:
Seems like what we looked up was for altering table. I get this error [Knex:warning - migrations failed with error: alter table "trip" add constraint "trip_tracknum_unique" unique ("tracknum") - column "tracknum" named in key does not exist
]
![2017-09-09 10 41 59](https://user-images.githubusercontent.com/28984604/30236004-db7304e2-954b-11e7-9669-a03d61ecb6dd.png)


I set `t.text('tracknum').unique();` for column instead and it's working.
![2017-09-09 10 41 01](https://user-images.githubusercontent.com/28984604/30235995-a5204bc0-954b-11e7-9864-2d64d6973141.png)